src/rewrite/steward.ts

```diff
@@ -6,16 +6,24 @@ import {
   VitalikStewardLegacy,
   LogBuy as LogBuyLegacy
 } from "../../generated/VitalikStewardLegacy/VitalikStewardLegacy";
+import {
+  GLOBAL_PATRONAGE_DENOMINATOR,
+  NUM_SECONDS_IN_YEAR_BIG_INT,
+  VITALIK_PATRONAGE_NUMERATOR,
+  VITALIK_PRICE_WHEN_OWNED_BY_SIMON
+} from "../CONSTANTS";
 // import { minBigInt } from "../util";
 
 function createDefaultPatron(address: Address, txTimestamp: BigInt): PatronNew {
   let addressString = address.toHexString();
   let patron = new PatronNew(addressString);
   patron.address = address;
   patron.totalTimeHeld = BigInt.fromI32(0);
+  patron.totalContributed = BigInt.fromI32(0);
+  patron.patronTokenCostScaledNumerator = BigInt.fromI32(0);
   patron.tokens = [];
   patron.lastUpdated = txTimestamp;
-  patron.save();
+  // patron.save();
```

Comment:
Can you explain why this is now commented out? :)

Reply:
Sure, I actually wanted to give an explanation on this PR anyway. Before I had to run off for food. I'll add it on the relevant lines in the code :)